types/oracledb/index.d.ts

```diff
@@ -2718,6 +3005,14 @@ declare namespace OracleDB {
          * @since 6.3
          */
         accessTokenConfig?: AccessTokenConfigAzure | AccessTokenConfigOCI | undefined;
+        /**
+         * An array of string values that identifies the application context used by the connection.
+         * The elements of the array should contain three string values - namespace, name, and value.
+         * Available only in node-oracledb Thin mode.
+         *
+         * @since 6.9
+         */
+        appContext?: Array<string>;
```

Comment:
appContext is an array of array elements, when each array element has 3 string values

Reply:
Updated.